R/render.R

```diff
@@ -324,6 +325,8 @@ add_links <- function(path, main_language = main_language,
       path_rel(path, output_folder, path_language, main_language)
     }
     href <- sprintf("%s/%s", site_url, new_path)
+    # If no translated version exists, don't add the link
```

Comment:
here you could use an explaining variable instead of the comment https://blog.r-hub.io/2023/01/26/code-comments-self-explaining-code/#use-helper-functions-or-explaining-variables

```r
no_translated_version <- (!fs::file_exists(file.path(output_folder, new_path)))
if (no_translated_version) return()

```

Reply:
same comment for the other place where this comes up